README.md

```diff
@@ -39,7 +39,8 @@ Keep up to date by following [@luckyframework](https://twitter.com/luckyframewor
 
 ## Documentation
 
-[API (main)](https://luckyframework.github.io/lucky/)
+- [API (main)](https://luckyframework.github.io/lucky/)
+- [HTTP/2 Support](guides/http_backend/09-http2-support.md) - Learn how to enable HTTP/2 in your Lucky app
```

Comment:
Not the right place for the docs (weird AI namespacing choice) but also didn’t feel right adding all these to the README.